packages/devtools/test/settings_controller_test.dart

```diff
@@ -35,9 +35,29 @@ void main() async {
       expect(sdkVersion, 'Flutter SDK Version: ${serviceManager.sdkVersion}');
 
       final flagList = await env.service.getFlagList();
+      expect(flags.length, flagList.flags.length);
+      final expectedFlags = [
+        Flag.parse({
+          'name': 'causal_async_stacks',
+          'comment': 'Improved async stacks',
+          'modified': true,
+          'valueAsString': 'true'
+        }),
+        Flag.parse({
+          'name': 'async_debugger',
+          'comment': 'Debugger support async functions.',
+          'modified': false,
+          'valueAsString': 'true'
+        }),
+      ];
       for (var i = 0; i < flags.length; i++) {
```

Comment:
The anyElement matches could make this a little simpler.

a slightly easier way to write this test is:
https://pub.dev/documentation/matcher/latest/matcher/anyElement.html
```dart
for (var expectedFlag in expectedFlags) {
  expect(flags, anyElement(equals(expectedFlag));
}
This is assuming the Flag class has implemented equals as well as toString.

Reply:
Yes but Flag is a code from vm_service and hasn't implemented equals yet so it's hard to write it this way, do you have any suggestion? Should I try to write the matcher by my own?